cmd/thanos/rule.go

```diff
@@ -7,7 +7,7 @@ import (
 	"bytes"
 	"context"
 	"fmt"
-	"html/template"
+	"text/template"
```

Comment:
I'm not sure if we also should change the other usages of "template"; i would prefer if we had "texttemplate" and "htmltemplate" and only changed the one invocation where html is problematic ( the url expression ).